src/pages/learning-journal.tsx

```diff
@@ -32,129 +50,192 @@ export async function getStaticProps () {
   }
 }
 
-type FormattedLearningJournal = Omit<LearningJournalType, 'date'> & {
-  dateTitle: string;
-}
+function LearningJournalContent () {
+  const [date, setDate] = useState<string>()
+  const [afterCursor, setAfterCursor] = useState<string>()
+  const [beforeCursor, setBeforeCursor] = useState<string>()
+
+  const { data: { edges, pageInfo } = {}, isValidating } = useSWR(
+    ['learning-journal-page', date, afterCursor, beforeCursor],
```

Comment:
I don't think the keys are working properly cause it always fetching pages from the same cursors even tho they have been requested before